src/main/java/frc/robot/DriverJoystick.java

```diff
@@ -95,11 +91,8 @@ private DriverJoystick() {
                 .onTrue(new InstantCommand(vision::enable0, vision))
                 .onFalse(new InstantCommand(vision::disable0, vision));
 
-        leftBumper.and(rightBumper.negate()).whileTrue(new LeftLiftUpCommand());
-        leftTrigger.and(rightTrigger.negate()).whileTrue(new LeftLiftDownCommand());
-
-        rightBumper.and(leftBumper.negate()).whileTrue(new RightLiftUpCommand());
-        rightTrigger.and(leftTrigger.negate()).whileTrue(new RightLiftDownCommand());
+        rightBumper.and(leftBumper.negate()).whileTrue(new LiftUpCommand());
```

Comment:
since left bumper/trigger is no longer used
```suggestion
        rightBumper.whileTrue(new LiftUpCommand());
```